File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    
    long longP = stol(p);
    for(int i = 0; i <= t.size() - p.size(); i++) {
        string temp = t.substr(i, p.size());
        long longT = stol(temp);
        if(longT <= longP) answer++;
    }
    
    return answer;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cs
using System;

public class Solution {
    public int solution(string t, string p) {
        int answer = 0;
        
        long intP = long.Parse(p);
        for(int i = 0; i <= t.Length - p.Length; i++) {
            long intT = long.Parse(t.Substring(i, p.Length));
            if(intT <= intP) answer++;
        }
        return answer;
    }
}